com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/expression/ExpressionGenerator.xtend

```diff
@@ -559,8 +555,8 @@ class ExpressionGenerator {
 			case FLATTEN: {
 				buildListOperationNoBody(op, "flattenList", params)
 			}
-			case DISTINCT, case ONLY_ELEMENT: {
-				distinctOrOnlyElement('''«op.receiver.javaCode(params)»''', op.operationKind === ListOperationKind.DISTINCT, op.operationKind === ListOperationKind.ONLY_ELEMENT)
+			case DISTINCT: {
+				distinctOrOnlyElement('''«op.receiver.javaCode(params)»''', true, false)
```

Comment:
Can the `distinctOrOnlyElement` method can be cleaned up to be just `distinct` method.  And no need for boolean parameters any more

Reply:
Done!